<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SamengesteldeLotnummers</class>
 <widget class="QDialog" name="SamengesteldeLotnummers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gebuikt lotnummer</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>391</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="tbLotnummer">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>40</y>
      <width>376</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>tbLotnummer</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbLotnummer">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Lotnummer:</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>90</y>
     <width>391</width>
     <height>236</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lbArtikelnummer">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>96</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Artikel:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbArtikelnummer">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>5</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="inputMask">
     <string>N.N.NN.NN</string>
    </property>
    <property name="text">
     <string>0.0.00.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbArtikelNaam">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>45</y>
      <width>376</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>tbArtikelNaam</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbHuidigVoorraad">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>145</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>tbHuidigVoorraad</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbNieuweVoorraad">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>205</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>tbNieuweVoorraad</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbBenodigdeHoeveelheid">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>175</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>De gebruikte hoeveelheid in deze samenstelling van dit specifiek lotnummer</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>tbBenodigdeHoeveelheid</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbHuidigVoorraad">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>150</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Huidig Voorraad:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbBenodigdeHoeveelheid">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Benodigde Hoeveelheid:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbNieuweVoorraad">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>116</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nieuwe Voorraad:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dtpInStock">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>75</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="dtpTGV">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>105</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbInStock">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>In Voorraad sinds:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbTGV">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>146</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Te gebruiken voor:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>330</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Opslaan</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Annuleren</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
